exercises/always_async/problem.md

```diff
@@ -14,7 +14,7 @@ promise will be called on the **next** turn of the event loop.
 In this lesson, we are going to prove to ourselves that this is the case
 by writing a script that does the following:
 
-1. Create a promise using the `q` library
+1. Create a promise
 2. Pass `console.log` to the `then` method of our promise
 3. Resolve the promise with a parameter `"SECOND"`
 4. Print `"FIRST"` to the console using `console.log`
```

Comment:
Here's "resolve" again.

Perhaps

```
Print `"FIRST"` to the console using `console.log` *immediately after* fulfilling the promise.
```

would make it easier to go from the instructions to the solution.